/home/dg9272/.conda/envs/cos485/lib/python3.10/site-packages/accelerate/accelerator.py:436: FutureWarning: Passing the following arguments to `Accelerator` is deprecated and will be removed in version 1.0 of Accelerate: dict_keys(['split_batches']). Please pass an `accelerate.DataLoaderConfiguration` instead: 
dataloader_config = DataLoaderConfiguration(split_batches=False)
  warnings.warn(
Detected kernel version 4.18.0, which is below the recommended minimum of 5.5.0; this can cause the process to hang. It is recommended to upgrade the kernel to the minimum version or higher.
Instantiating the dataset and dataloader...
Checking the dataset and dataloader...
person_images: torch.Size([1, 3, 96, 96])
ca_images: torch.Size([1, 3, 96, 96])
garment_images: torch.Size([1, 3, 96, 96])
person_poses: torch.Size([1, 25, 2])
garment_poses: torch.Size([1, 1, 2])
Instantiating U-Nets...
((96, 96),)
Instantiating the trainer...
Distributed environment: NO
Num processes: 1
Process index: 0
Local process index: 0
Device: cuda

Mixed precision type: no

Traceback (most recent call last):
  File "/home/dg9272/tryondiffusion2.0/test.py", line 154, in <module>
    main()
  File "/home/dg9272/tryondiffusion2.0/test.py", line 116, in main
    trainer = TryOnImagenTrainer(
  File "/home/dg9272/tryondiffusion2.0/tryondiffusion/tryon_imagen_trainer.py", line 261, in __init__
    self.load_from_checkpoint_folder(init_checkpoint_path=init_checkpoint_path)
  File "/home/dg9272/tryondiffusion2.0/tryondiffusion/tryon_imagen_trainer.py", line 527, in load_from_checkpoint_folder
    sorted_checkpoints = self.all_checkpoints_sorted
  File "/home/dg9272/tryondiffusion2.0/tryondiffusion/tryon_imagen_trainer.py", line 514, in all_checkpoints_sorted
    sorted_checkpoints = sorted(checkpoints, key=lambda x: int(str(x).split(".")[-2]), reverse=True)
  File "/home/dg9272/tryondiffusion2.0/tryondiffusion/tryon_imagen_trainer.py", line 514, in <lambda>
    sorted_checkpoints = sorted(checkpoints, key=lambda x: int(str(x).split(".")[-2]), reverse=True)
ValueError: invalid literal for int() with base 10: '/scratch/network/dg9272/cos485/checkpoints/10k_check_baseline'
